src/block-management/create-block-and-log-streamer.js

```diff
@@ -57,17 +56,30 @@ function createBlockAndLogStreamer(configuration, transport) {
     });
 
     internalState.setState({ blockAndLogStreamer: blockAndLogStreamer, blockNotifier: blockNotifier });
+
+    callback = callback ? callback : function (e) { if (e) console.log(e); };
```

Comment:
Suggest checking that the callback is actually a function (rather than just checking for existence):
```javascript
callback = typeof callback === "function" ? callback : function (e) { if (e) console.log(e); };
```

Reply:
Can is isFunction, and even get the out of band handler out of state...